app/main/routes_picoferm_api.py

```diff
@@ -71,10 +71,17 @@ def process_picoferm_firmware(file):
 @main.route('/API/PicoFerm/getState')
 @use_args(get_ferm_state_args, location='querystring')
 def process_get_ferm_state(args):
-    if args['uid'] not in active_ferm_sessions:
-        create_new_session(args['uid'])
+    uid = args['uid']
+    if uid not in active_ferm_sessions:
+        active_ferm_sessions[uid] = PicoFermSession()

```

Comment:
method `create_new_session(uid)` actually creates the session in memory and as a file, so here just creating an in-memory session as to be aligned with other devices' eager session creation logic